package.json

```diff
@@ -188,7 +188,7 @@
   "dependencies": {
     "fabric8-analytics-lsp-server": "0.1.48",
     "fs": "0.0.1-security",
-    "path": "^0.12.7",
+    "path": "0.12.7",
```

Comment:
don't use npm package `path` nodejs provide that option

Reply:
yes am aware of that, used it as I took https://github.com/redhat-developer/vscode-adapters/blob/master/package.json#L244 as reference ;)